js/game.js

```diff
@@ -121,7 +136,7 @@ function game_do_gravity(state) {
 }
 
 function game_try_move_block(state, y, x, dy, dx, slide_rule) {
-    // (State, int, int, int, int, SLIDE_) -> bool
+    // (State, int, int, int, int, SLIDE_, bool) -> bool
```

Comment:
`slide_rule` is not a `bool`, it's one of `SLIDE_NONE`, `SLIDE_MOVED`, or `SLIDE_ALL`